data/faarish.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "Salman Farish",
+  "bio": "Aspiring Open Source Developer | Tech Enthusiast"
+}
```

There's a missing comma and a trailing bracket here! 
```suggestion
  "bio": "Aspiring Open Source Developer | Tech Enthusiast",
```